README.md

```diff
@@ -547,6 +547,23 @@ sent via this stream.
   client.stop_stream(cancel_requests=True)
 ```
 
+For AsyncIO requests, an AsyncIO task wrapping an `infer()` coroutine can be
```

Comment:
Let's call it gRPC AsyncIO to be more explicit?

Reply:
Sure, updated.